docs/products/SmartBear MCP Server/zephyr-integration.md

```diff
@@ -20,3 +20,8 @@ The following environment variables configure the Zephyr integration:
 
 - **Purpose**: Retrieve projects available within your Zephyr account.
```

Comment:
I know is not part of your PR but you think we could say that we return only the projects the user has JIRA read access, or we prefer not to put such kind of information as it can change in the future?